src/response/phonon.jl

```diff
@@ -0,0 +1,291 @@
+using Unitful
+using UnitfulAtomic
+
+struct P6{T <: Real}
+    coordinate::Vec3{T}     # Fractional coordinate of `q`-point
+    # TODO: see if we use this
+    n_spin_components::Int  # 2 if :collinear, 1 otherwise
+
+    # `kpt.coordinate + q = kpt′.coordinate + ΔG`
+    # TODO: see if we use this
+    mapping::Vector{Int}           # Index of ``k``-point `kpt′`
+    ΔG_vectors::Vector{Vec3{Int}}  # Shifts ΔG ∈ ℤ³
+
+    ordering
+end
+
+QPoint = P6
```

Comment:
I'm not sure we really need this to be in a struct. The response functions just need the q coordinate, and the precomputed stuff can be done at a lower level